### PR TITLE
📝 Add a features page & re-organize landing page

### PR DESCRIPTION
**Pull request by falexwolf**

### CODE REVIEW

File: docs/features-lamindb.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+- Zero lock-in: universal, simple storage formats, LaminDB is **not** a client for "Lamin Cloud" but can run server-side allowing you to build your own apps
+- Scalable: metadata tables support 100s of millions of entries at reasonable query speed, otherwise Lamin relies on object stores
+- Flexible storage backends (local, S3, GCP, anything `fsspec` supports)
+- Currently two SQL backends: SQLite & Postgres (more to come)
+- Fine-grained access management via embedded storage & SQL restrictions & high-level access management through Lamin's collaborator roles
+- Secure: fully secure & embedded in your infrastructure & Lamin has no access to your data & metadata
+- Idempotent & ACID operations
+- File & dataset versioning
+- Numerous safeguards against typos & duplications when populating
+- A small number of high-quality dependencies
```
Comment (Zethson) [inline docs/features-lamindb.md:34]:
```suggestion
```

Too technical.

File: docs/features-lamindb.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  - access: {class}`~lamindb.dev.Registry.filter`, {class}`~lamindb.dev.Registry.search`, {class}`~lamindb.File.stage`, {class}`~lamindb.File.backed`, {class}`~lamindb.File.load`
+  - {class}`~lamindb.dev.CanValidate.validate` & {class}`~lamindb.dev.CanValidate.standardize` metadata of vectors & arrays, {class}`~lamindb.dev.CanValidate.inspect` validation failures
+  - register & annotate: use {class}`~lamindb.dev.LabelManager.add` to annotate with untyped or typed labels, {class}`~lamindb.File.save` to save data & metadata
+- {meth}`~lamindb.track` data flow (provenance/ lineage) across notebooks, pipelines & UI
+- Manage ontologies, embed custom ontologies in public knowledge
+- Model data schema-less or schema-full, add custom schema plug-ins, manage schema migrations
+- No silos: create DB instances within seconds and share data across a mesh of instances
+- Awareness of common array formats in memory & storage: `DataFrame`, `AnnData`, `MuData`, `pyarrow.Table` backed by `parquet`, `zarr`, `TileDB`, `HDF5`
+  - `.backed()` provides access to connectors that allow stream-based queries
+  - `.join()` allows auto-concatenating DataFrames
```
Comment (Zethson) [inline docs/features-lamindb.md:21]:
```suggestion
```

Too boring/technical?

File: docs/features-lamindb.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+**How?**
+
+- Zero lock-in: universal, simple storage formats, LaminDB is **not** a client for "Lamin Cloud" but can run server-side allowing you to build your own apps
+- Scalable: metadata tables support 100s of millions of entries at reasonable query speed, otherwise Lamin relies on object stores
+- Flexible storage backends (local, S3, GCP, anything `fsspec` supports)
```
Comment (Zethson) [inline docs/features-lamindb.md:27]: Maybe link to fsspec? This is pretty technical and not everyone knows it.

File: docs/features-lamindb.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+- Currently two SQL backends: SQLite & Postgres (more to come)
+- Fine-grained access management via embedded storage & SQL restrictions & high-level access management through Lamin's collaborator roles
+- Secure: fully secure & embedded in your infrastructure & Lamin has no access to your data & metadata
+- Idempotent & ACID operations
```
Comment (Zethson) [inline docs/features-lamindb.md:31]: Link to the corresponding sections. Think we had that in the FAQ?

Comment (falexwolf) [reply docs/features-lamindb.md:31]: Agreed, was already about to add links to each statement!

File: docs/features-lamindb.md (hunk 1 above)
Comment (falexwolf) [reply docs/features-lamindb.md:34]: OK, will remove!

File: docs/features-lamindb.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+**What?**
+
+- Unified API to manage data in storage & metadata in SQL database
+  - access: {class}`~lamindb.dev.Registry.filter`, {class}`~lamindb.dev.Registry.search`, {class}`~lamindb.File.stage`, {class}`~lamindb.File.backed`, {class}`~lamindb.File.load`
+  - {class}`~lamindb.dev.CanValidate.validate` & {class}`~lamindb.dev.CanValidate.standardize` metadata of vectors & arrays, {class}`~lamindb.dev.CanValidate.inspect` validation failures
+  - register & annotate: use {class}`~lamindb.dev.LabelManager.add` to annotate with untyped or typed labels, {class}`~lamindb.File.save` to save data & metadata
+- {meth}`~lamindb.track` data flow (provenance/ lineage) across notebooks, pipelines & UI
+- Manage ontologies, embed custom ontologies in public knowledge
+- Model data schema-less or schema-full, add custom schema plug-ins, manage schema migrations
```
[the remark this answers is not in the document]
Comment (falexwolf) [reply docs/features-lamindb.md:17]: OK!

[the remark this answers is not in the document]
Comment (falexwolf) [reply docs/features-lamindb.md:17]: Done!

File: docs/features-lamindb.md (hunk 2 above)
Comment (falexwolf) [reply docs/features-lamindb.md:21]: OK, will remove for now.

File: docs/features-lamindb.md (hunk 3 above)
Comment (falexwolf) [reply docs/features-lamindb.md:27]: Done!

File: docs/features-lamindb.md (hunk 4 above)
Comment (falexwolf) [reply docs/features-lamindb.md:31]: Done!

File: docs/features-lamindb.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+- No silos: create DB instances within seconds and share data across a mesh of instances
```
[the remark this answers is not in the document]
Comment (falexwolf) [reply docs/features-lamindb.md:19]: I feel this is important. And I feel it's OK if this list is a bit longer. But I'm sure we'll keep discussing the right granularity.

**Comment by sunnyosun** [comment]
An idea for general structure: it's a bit hard for me to read through what and how and connect them. What about having a dropdown of how under each what bullet? So users can expand to see how when the read through the bullet of features.

You could then even remove the "What" heading as that's the list of features.

**Pull request merged**